src/test/unit/regions/TMRegionTest.cpp

```diff
@@ -213,7 +213,7 @@ TEST(TMRegionTest, testLinking) {
   std::shared_ptr<Region> region1 = net.addRegion("region1", "VectorFileSensor",parameters);
   std::shared_ptr<Region> region2 = net.addRegion("region2", "SPRegion", "{dim: [2,10]}");
   std::shared_ptr<Region> region3 = net.addRegion("region3", "TMRegion",
-                                        "{activationThreshold: 9, cellsPerColumn: 5}");
+                                        "{activationThreshold: 11, cellsPerColumn: 5}");
```

Comment:
no problem, but why these changes?

Reply:
>  why these changes?

This test is to make sure that a field can be set to something other than the default.  I had to change the values for activationThreshold in this tesst because it had to be larger than the minimum which is now 10.